### PR TITLE
[VerticalGalleries] vertical gallery component

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What

Creates base component for the verticalGallery
# Why

Introduces new base component for the new verticalGallery experience

https://skype.visualstudio.com/SPOOL/_workitems/edit/3151598
# How Tested

Introduction of base component, responsive wrapper and storybook page to come in later PR's

### CODE REVIEW

File: packages/react-components/src/components/VerticalGallery.tsx [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+  childrenContainerStyle,
+  controlBarContainerStyle,
+  leftRightButtonStyles,
+  rootStyle
+} from './styles/VerticalGallery.styles';
+
+export interface VerticalGalleryStyles extends BaseCustomStyles {
+  /** Styles for each video tile in the vertical gallery */
+  children?: IStyle;
+  /** Styles for the previous button in the verticalGallery control bar */
```
Comment (edwardlee-msft) [inline packages/react-components/src/components/VerticalGallery.tsx:18]: Does this not style the next button as well?

Comment (dmceachernmsft) [reply packages/react-components/src/components/VerticalGallery.tsx:18]: forgot to update the comment from my initial idea

File: packages/react-components/src/components/VerticalGallery.tsx [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+  const showButtons = numberOfChildren > childrenPerPage;
+
+  const onPreviousButtonClick = () => {
+    setPage(page - 1);
+  };
+  const onNextButtonClick = () => {
+    setPage(page + 1);
+  };
+
+  useEffect(() => {
```
Comment (edwardlee-msft) [inline packages/react-components/src/components/VerticalGallery.tsx:96]: Would this useEffect run when large amounts of people leave a call? 
Resulting in lower numberOfChildren?

**Review by mgamis-msft — APPROVED** [review]

File: common/config/babel/.babelrc.js [hunk 3]
```diff
@@ -46,7 +46,9 @@ process.env['COMMUNICATION_REACT_FLAVOR'] === 'stable' &&
         // feature for tracking environment info API different than unsupported browser. stable use of environment info affects other components possibly sooner
         'calling-environment-info',
         // feature for blurred background and replace background effects
-        'video-background-effects'
+        'video-background-effects',
+        // feature for new vertical gallery layouts in videoGallery
```
Comment (edwardlee-msft) [inline common/config/babel/.babelrc.js:50]: nit: feature for vertical gallery layouts in VideoGallery

File: packages/react-components/src/components/styles/VerticalGallery.styles.ts [hunk 4]
```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { IStyle, Theme } from '@fluentui/react';
+
+/**
+ * Horizontal Gallery gap size in rem between tiles and buttons
+ */
```
Comment (edwardlee-msft) [inline packages/react-components/src/components/styles/VerticalGallery.styles.ts:8]: Not private?

**Review by edwardlee-msft — APPROVED** [review]

**Pull request merged**